app/views/spree/admin/products/new.html.haml

```diff
@@ -50,7 +50,7 @@
             = f.label :price, t(".price")
             %span.required *
             %br/
-            = f.text_field :price, { "class" => "fullwidth", "ng-model" => "product.price" }
+            = f.text_field :price, { "class": "fullwidth", "ng-model": "product.price", "ng-init": "product.price='#{@product.price}'" }
```

Comment:
I tested with it, and it was renderer as:

```html
<input class="fullwidth" ng="model product.price value '1.0'" type="text" value="1.0" name="product[price]" id="product_price">
```
That's why spec did fail (🙏 )

My conclusion was then it works when you use `%input` like this:
```haml
%input{ type: 'text', name: 'firstname', required: true, ng: { model: 'address.firstname'} }
```

but not when using `f.text_field`
Did I already tell you that I don't link AngularJS? :trollface: 

Reverted your suggestion then, sorry.

Reply:
Ah, I see. The nested attributes are a HAML feature but here we are giving options to a Rails helper. That's different, sorry. Seeing this code I was wondering how much simpler it could be without AngularJS...